example/streams/filebeat.yml

```diff
@@ -0,0 +1,28 @@
+#filebeat.config:
+#  prospectors:
+#    path: ${path.config}/prospectors.d/*.yml
+#    reload.enabled: false
+#  modules:
+#    path: ${path.config}/modules.d/*.yml
+#    reload.enabled: false
```

Comment:
Argh, missed to remove these commented-out lines. I'll do so in follow-up commits.